.github/workflows/build-cmake.yaml

```diff
@@ -91,19 +91,18 @@ jobs:
         run: |
           sudo apt update -y
           sudo apt install -y \
-              ninja-build \
+              clang-19 \
```

Comment:
Why does it differ from docs/INSTALL.md?

Reply:
- clang is optional to build OM on Linux
- gcc/g++ is the main compiler on Linux, it is installed with `build-essential` package on Ubuntu
- without explicitly specifying `-stdlib=libc++`, clang uses _gcc libstdc++_, that leads to issues like here (fixed in clang-19).